src/define_plugin_sets.h

```diff
@@ -826,34 +826,34 @@ To create/register a plugin, you have to :
     #define USES_P064   // APDS9960
     #define USES_P065   // DRF0299
     #define USES_P066   // VEML6040
-    #define USES_P067   // HX711_Load_Cell
+    // #define USES_P067   // HX711_Load_Cell
     #define USES_P068   // SHT3x
     #define USES_P069   // LM75A
 
-    #define USES_P070   // NeoPixel_Clock
-    #define USES_P071   // Kamstrup401
-    #define USES_P072   // HDC1080
-    #define USES_P074   // TSL2561
+    //#define USES_P070   // NeoPixel_Clock
+    //#define USES_P071   // Kamstrup401
+    //#define USES_P072   // HDC1080
+    //#define USES_P074   // TSL2561
```

Comment:
Removing plugins from these defines does mess up the build configuration.
We do need to redefine the plugin sets, but not like this.
If it doesn't fit due to font sizes, we should consider making a new set of build definitions.

See: https://github.com/letscontrolit/ESPEasy/issues/2590

Reply:
I had no luck in compiling with a lot of plugins since the firmware was too large and resulted in an error. How can I compile on a custum build with only the plugins i need ?
Therefore i reduces the plugins :)

Reply:
Use a Custom.h file (sample is included) and select the desired custom environment to compile using PlatformIO.
For example in VScode:
![image](https://user-images.githubusercontent.com/3751318/84120833-a9af1100-aa36-11ea-88c2-26b79421c419.png)

Reply:
great ! Switched from Atom now to Visual Code, thx